File: CS/Step 1 - creating model/Commands/IMenuCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Input;
using System.Windows.Media;
using System.Windows.Data;
using DevExpress.Xpf.Bars.Native;

namespace DXBarsAndMVVM.Commands {
    public interface IMenuCommand {
        string Caption { get; }
        Uri Glyph { get; }
        Binding Binding { get; }
        ICommand Command { get; }
        object CommandParameter { get; }
        
    }
}

File: VB/Step 1 - creating model/Commands/IMenuCommand.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Input
Imports System.Windows.Media
Imports System.Windows.Data
Imports DevExpress.Xpf.Bars.Native

Namespace DXBarsAndMVVM.Commands
    Public Interface IMenuCommand
        ReadOnly Property Caption() As String
        ReadOnly Property Glyph() As Uri
        ReadOnly Property Binding() As Binding
        ReadOnly Property Command() As ICommand
        ReadOnly Property CommandParameter() As Object

    End Interface
End Namespace
